.github/workflows/main.yml

```diff
@@ -29,7 +29,16 @@ jobs:
         xcode:
         - '14.2' # Swift 5.7
         - '14.3' # Swift 5.8
+        - '15.0' # Swift 5.9
     steps:
       - uses: actions/checkout@v2
       - name: Test Package Plugin
-        run: swift package --allow-writing-to-package-directory format --lint
+        run: swift package --allow-writing-to-package-directory format --lint --log
+
+  unit-tests:
```

Comment:
We can keep this code in mind if we want to add unit tests to this project in the future.